tools/buildsys/Cargo.toml

```diff
@@ -12,7 +12,7 @@ exclude = ["README.md"]
 duct = "0.13.0"
 hex = "0.4.0"
 rand = { version = "0.8", default-features = false, features = ["std", "std_rng"] }
-reqwest = { version = "0.10.1", default-features = false, features = ["rustls-tls", "blocking"] }
+reqwest = { version = "0.11.1", default-features = false, features = ["rustls-tls", "blocking"] }
 serde = { version = "1.0", features = ["derive"] }
```

Comment:
even `0.11.2` is available, are we not updating directly to that ? Or you are ignoring here because minor version change does not matter and `Cargo.lock` will include `0.11.2`

Any reason why we are not removing minor version and change it to "0.11" for all new dependency updates ? We know minor version any way does not matter.

Reply:
If the micro version is specified, it can generally be taken to mean something like "this is the minimum version we've tested with, so you shouldn't use 0.11.0 because we can't guarantee it'll work, but later versions probably will work."